src/views/edit-catalog/select-form.tsx

```
@@ -11,6 +13,16 @@ import { Button } from "~/shared/ui/button";
 
 import useCatalogStore from "./catalog-store";
 
+function formatSubscribers(subs: number) {
+  const K = 1_000;
+  const M = 1_000_000;
+
+  if (subs > M) return (subs / M).toFixed(2) + "M";
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)